homeworks/vancsikpeti/4_exceptions_logging/to_do.py

```diff
@@ -0,0 +1,124 @@
+import os
+import logging
+
+file_handler = logging.FileHandler("python_hu/homeworks/vancsikpeti/4_exceptions_logging/todo.log")
+fformatter = logging.Formatter('%(asctime)s - %(name)s - %(module)s - %(levelname)s - %(message)s')
+file_handler.setFormatter(fformatter)
+file_handler.setLevel(logging.DEBUG)
+
+stream_handler = logging.StreamHandler()
+sformatter = logging.Formatter('%(levelname)s - %(message)s - %(asctime)s')
+stream_handler.setFormatter(sformatter)
+stream_handler.setLevel(logging.ERROR)
+
+logger = logging.getLogger()
+logger.addHandler(file_handler)
+logger.addHandler(stream_handler)
+logger.setLevel(logging.DEBUG)
+
+file_path = "python_hu/homeworks/vancsikpeti/4_exceptions_logging/to-do-list.txt"
+with open(file_path, "w") as file:
+    file.write("")
+    logger.info("Create file.")
+
+def view_task():
+    with open(file_path, "r") as file:
```

Comment:
Ezt a context manager-t olvasásra 2 helyen is használod (itt, és a `remove_tasks()`-ban).
Szuper lenne ezt a logikát átrakni egy külön function-be, és ott try-except-ttel lekezelni a különböző lehetséges hibákat 
(Filenotfound, IO, OS). Így meg tudod hívni 1 soron ezeken a helyeken a function-t.